aptos-move/aptos-vm/src/move_vm_ext/session.rs

```diff
@@ -36,6 +36,13 @@ use std::{
     sync::Arc,
 };
 
+pub(crate) enum ResourceGroupChangeSet {
+    // Merged resource groups op.
+    V0(HashMap<StateKey, MoveStorageOp<Bytes>>),
+    // Granular ops to individual resources within a group.
+    V1(HashMap<StateKey, AccountChangeSet>),
```

Comment:
why this is AccountChangeSet? we don't expect modules to be there right? a BtreeMap<StructTag, Op\<Bytes\>> seems cleaner

Reply:
+1

Reply:
I think I kept using AccountChangeSet because it was there before, good call out.